daemon/mgr/container_state.go

```diff
@@ -133,4 +137,33 @@ func (c *Container) SetStatusUnpaused() {
 	c.Lock()
 	defer c.Unlock()
 	c.State.Status = types.StatusRunning
+	c.setStatusFlags(types.StatusRunning)
```

Comment:
I just wonder why use a function instead of a single line like `c.State.Running=true` ?

Reply:
we have 4 flags : Dead, Paused, Restarting and Running , the flags are exclusive，so when set one flag, we should unset others. WDYT? @Ace-Tang